exercises/forth/forth_test.go

```diff
@@ -0,0 +1,63 @@
+package forth
+
+// API:
+// func Forth([]string) ([]int, error)
+//
+
+import (
+	"reflect"
+	"testing"
+)
+
+const targetTestVersion = 1
+
+type testCase struct {
+	description string
+	input       []string
+	expected    []int // nil slice indicates error expected.
+}
+
+type testcaseSection struct {
+	name  string
+	tests *[]testCase
+}
+
+func TestTestVersion(t *testing.T) {
+	if testVersion != targetTestVersion {
+		t.Fatalf("Found testVersion = %v, want %v", testVersion, targetTestVersion)
+	}
+}
+
+func runTestCases(t *testing.T, sectionName string, testCases *[]testCase) {
+	for _, tc := range *testCases {
+		if v, err := Forth(tc.input); err == nil {
+			var _ error = err
+			if tc.expected == nil {
+				t.Fatalf("%s - %s\n\tForth(%#v) expected an error, got %v",
```

Comment:
when you make your change to the prefixes, you may notice that you are repeating the `%s - %s\n\tForth(%#v)` a lot. you could consider extracting it to a `prefix :=` or something, so you only have to say it once.